src/iterative_sorting/iterative_sorting.py

```diff
@@ -18,7 +22,10 @@ def selection_sort(arr):
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort(arr):
     # Your code here
-
+    for i in range(len(arr) - 1): 
+        for n in range(len(arr) - 1):
```

Comment:
Could shorten this to not go over sorted part of list again, but this is fine either way,.